deploy-kovan-dryrun.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I think we should rename this file to `prepare-kovan-like.sh` or something similar.